src/components/BookDropDown/BookDropDown.js

```diff
@@ -0,0 +1,205 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import { Button, Typography } from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import PropTypes from 'prop-types';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Box from '@material-ui/core/Box';
+import { API } from '../../store/api';
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role='tabpanel'
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box p={3}>{children}</Box>}
+    </div>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+const useStyles = makeStyles((theme) => ({
+  bookButton: {
+    padding: '0 8px',
+    width: 71,
+  },
+  bookCard: {
+    minWidth: 600,
+  },
+  chapterButton: {
+    padding: 0,
+    width: 36,
+    minWidth: 'unset',
+  },
+}));
+
+export default function BookDropDown(props) {
+  const classes = useStyles();
+  const { value, onChange, width, selectProject } = props;
+  const [booksValue, setBooksValue] = useState([]);
+  const [open, setOpen] = React.useState(false);
+  const [book, setBook] = React.useState([]);
+  const [chapter, setChapter] = React.useState(1);
+
+  useEffect(() => {
+    if (selectProject !== undefined) {
+      API.get(
+        `/autographa/project/versification?project_id=${selectProject.projectId}`
```

Comment:
This will throw error if no project is selected, we can put a check if selectedProject is null or undefined,